use_cases/README.md

```diff
@@ -4,6 +4,9 @@ The goal of this guide is a central location for all tutorials on solving use ca
 
 ## Troubleshooting errors and issues
 
+## ECS FireLens New Features
+* [Multiple Config support](init-process-for-fluent-bit/README.md)
```

Comment:
Let's make this a better advert to catch users attention... like "Source multiple configs from S3 or files"